Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -1383,6 +1454,42 @@ namespace PhysX
         return m_shapeConfiguration.m_capsule.m_height;
     }
 
+    void EditorColliderComponent::SetCylinderRadius(float radius)
+    {
+        m_shapeConfiguration.m_cylinder.m_radius = radius;
+        UpdateCylinderCookedMesh();
+        CreateStaticEditorCollider();
+    }
+
+    float EditorColliderComponent::GetCylinderRadius()
+    {
+        return m_shapeConfiguration.m_cylinder.m_radius;
+    }
+
+    void EditorColliderComponent::SetCylinderHeight(float height)
+    {
+        m_shapeConfiguration.m_cylinder.m_height = height;
+        UpdateCylinderCookedMesh();
+        CreateStaticEditorCollider();
+    }
+
+    float EditorColliderComponent::GetCylinderHeight()
```

Comment:
suggestion: `const` again (even if this is an EBus it's possible to do this now and likely preferred, not a huge deal but always good to be const correct 🙂 )

Reply:
yeah the rest of the getters in that EBus were non-const for some reason too, now changed all of them!

Reply:
Legend! 😄 Thanks! 👍

Reply:
(and there's always `mutable` if we need it 😉)